last_modified.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/Ghidra/Framework/Utility/src/main/java/generic/jar/ResourceFile.java b/Ghidra/Framework/Utility/src/main/java/generic/jar/ResourceFile.java
+index 1b8f3059e..0982f3a1d 100644
+--- a/Ghidra/Framework/Utility/src/main/java/generic/jar/ResourceFile.java
++++ b/Ghidra/Framework/Utility/src/main/java/generic/jar/ResourceFile.java
+@@ -31,6 +31,7 @@ public class ResourceFile implements Comparable<ResourceFile> {
+ 	private static final String JAR_FILE_PREFIX = "jar:file:";
+ 	private Resource resource;
+ 	private static Map<String, JarResource> jarRootsMap = new HashMap<String, JarResource>();
++	private long lastModifiedTime;
```

Comment:
Can you please file a PR against Ghidra and reference it in the commit message?

Reply:
You mean to create a pull request on Ghidra's git ([NationalSecurityAgency/ghidra](https://github.com/NationalSecurityAgency/ghidra))? I don't think they will accept. Trying anyway.

Reply:
Even if your PR isn't accepted, at least we can use it to link to solutions that were merged, or an explanation as to why it's still a downstream patch, something which would be lost in the review in this PR.

Reply:
Ok

Reply:
Pull request: NationalSecurityAgency/ghidra#4107

Reply:
Done: 56fdbe1f76d8890f7ec60deb44fe0b9f8edecad9